pages/common/nnn.md

```diff
@@ -0,0 +1,32 @@
+# nnn
+
+> The missing terminal file manager for X.
+> Read more at https://github.com/jarun/nnn.
+
+- Specify bookmark key to open:
+
+`nnn -b {{key}}`
+
+- Show hidden files:
+
+`nnn -d`
+
+- Start in light mode (fewer details):
+
+`nnn -l`
+
+- Use version compare to sort files:
+
+`nnn -n`
+
+- Copy (or pick) selection to file, or stdout if file='-':
```

Comment:
I'm not really sure what this command does, could you provide an example usage? It does not seem to do what's described here (at least for what I tried).

Reply:
It just copies the selected file path and outputs it to stdout or a specifyd file.

try: `nnn -p -` and then select an file and press enter

Reply:
example of commando + output after I've selected the file "7zr.md"

<img width="417" alt="image" src="https://user-images.githubusercontent.com/10723098/59091104-dc4b9880-890e-11e9-8f6f-f6a5f2ca8652.png">

Reply:
@chrillebile thank you for the explanation!